modules/integration/tests/src/test/java/org/wso2/appmanager/integration/utils/bean/AppCreateRequest.java

```diff
@@ -0,0 +1,1174 @@
+     * @param oauthApiConsumerKey2 String.
+     */
+    public void setOauthApiConsumerKey2(String oauthApiConsumerKey2) {
+        this.oauthApiConsumerKey2 = oauthApiConsumerKey2;
+    }
+
+    /**
+     * Get Oauth Api Consumer Secret2.
+     * @return oauthApiConsumerSecret2 String.
+     */
+    public String getOauthApiConsumerSecret2() {
+        return oauthApiConsumerSecret2;
+    }
+
+    /**
+     * Set Oauth Api Consumer Secret2.
+     * @param oauthApiConsumerSecret2 String.
+     */
+    public void setOauthApiConsumerSecret2(String oauthApiConsumerSecret2) {
+        this.oauthApiConsumerSecret2 = oauthApiConsumerSecret2;
+    }
+
+    /**
+     * Get Oath Api Name2.
+     * @return oauthApiName2 String.
+     */
+    public String getOauthApiName2() {
+        return oauthApiName2;
+    }
+
+    /**
+     * Set Oath Api Name2.
+     * @param oauthApiName2
+     */
+    public void setOauthApiName2(String oauthApiName2) {
+        this.oauthApiName2 = oauthApiName2;
+    }
+
+    /**
+     * Get Oauth Api Token Endpoint3.
+     * @return oauthApiTokenEndpoint3 String.
+     */
+    public String getOauthApiTokenEndpoint3() {
+        return oauthApiTokenEndpoint3;
+    }
+
+    /**
+     * Set Oauth Api Token Endpoint3.
+     * @param oauthApiTokenEndpoint3 String.
+     */
+    public void setOauthApiTokenEndpoint3(String oauthApiTokenEndpoint3) {
+        this.oauthApiTokenEndpoint3 = oauthApiTokenEndpoint3;
+    }
+
+    /**
+     * Get Oauth Api Consumer Key3.
+     * @return oauthApiConsumerKey3 String.
+     */
+    public String getOauthApiConsumerKey3() {
+        return oauthApiConsumerKey3;
+    }
+
+    /**
+     * Set Oauth Api Consumer Key3.
+     * @param oauthApiConsumerKey3 String.
+     */
+    public void setOauthApiConsumerKey3(String oauthApiConsumerKey3) {
+        this.oauthApiConsumerKey3 = oauthApiConsumerKey3;
+    }
+
+    /**
+     * Get Oauth Api Consumer Secret3.
+     * @return oauthApiConsumerSecret3 String.
+     */
+    public String getOauthApiConsumerSecret3() {
+        return oauthApiConsumerSecret3;
+    }
+
+    /**
+     * Set Oauth Api Consumer Secret3.
+     * @param oauthApiConsumerSecret3 String.
+     */
+    public void setOauthApiConsumerSecret3(String oauthApiConsumerSecret3) {
+        this.oauthApiConsumerSecret3 = oauthApiConsumerSecret3;
+    }
+
+    /**
+     * Get Oath Api Name3.
+     * @return oauthApiName3 String.
+     */
+    public String getOauthApiName3() {
+        return oauthApiName3;
+    }
+
+    /**
+     * Set Oath Api Name3.
+     * @param oauthApiName3
+     */
+    public void setOauthApiName3(String oauthApiName3) {
+        this.oauthApiName3 = oauthApiName3;
+    }
+
+    /**
+     * Get Web app.
+     * @return webapp String.
+     */
+    public String getWebapp() {
+        return webapp;
+    }
+
+    /**
+     * Set Web App.
+     * @param webapp String.
+     */
+    public void setWebapp(String webapp) {
+        this.webapp = webapp;
+    }
+
```

Comment:
Fixed in https://github.com/wso2/product-app-manager/commit/f021f86c3c398dddeeca17e225a37e42f7f7033b